algorithms/prims-algorithm/prims-algorithm.tex

```diff
@@ -0,0 +1,63 @@
+\section{Prim's Algorithm}\index{graph!Prim's Algorithm}
+#ifdef hackpackpp
+Prim's algorithm is a greedy algorithm that allows for finding a minimum spanning tree/forest in $O(E + \log{V})$ time.
+Prim's algorithm is implemented by doing the following:
+
+\begin{enumerate}
+	\item Choose an arbitrary unvisited node in the graph.
+	\item Grow the tree by choosing the node that can be added to the tree by least cost.
+	\item Repeat step 2 until all connected nodes have been visited. If there are still nodes, return to step 1.
+\end{enumerate}
+
+#endif
+\subsection{Applications}
+\begin{itemize}
+	\item Finding a minimum spanning tree
+\end{itemize}
+
+#ifdef hackpackpp
+\subsection{Example Contest Problem: Cow Connection}
+Farmer John has not been very diligent in mowing his farm.
+As a result, the grazing field now resembles a maze with small openings at the intersections.
+
+At night, this maze is unlit, and is very dangerous.
```

Comment:
The second comma needs to die. 'is very dangerous' is not an independent clause (haha). Or you can make it '_it_ is very dangerous'.